core_v2/sources/domains.move

```diff
@@ -1108,6 +1108,39 @@ module aptos_names_v2::domains {
         (extract_subdomain_name(record), record.domain_name)
     }
 
+    #[view]
+    public fun get_name_record_v1_props_for_domain(
+        domain: String,
+    ): (u64, Option<address>) acquires CollectionCapabilityV2, NameRecordV2 {
+        get_name_record_v1_props_for_name(option::none(), domain)
+    }
+
+    #[view]
+    public fun get_name_record_v1_props_for_subdomain(
+        subdomain: String,
+        domain: String,
+    ): (u64, Option<address>) acquires CollectionCapabilityV2, NameRecordV2 {
+        get_name_record_v1_props_for_name(option::some(subdomain), domain)
+    }
+
+    #[view]
+    public fun get_reverse_lookup_name(
+        account_addr: address
+    ): Option<String> acquires ReverseRecord, NameRecordV2 {
```

Comment:
this will return `hello.apt` if lookup result is a domain, instead of just `hello`